stock_weighing_threaded_print/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Speed up label printing.
```

Comment:
This is not accurate. It should say that it allows to return the control of the user screen sooner, triggering the printing in a separate process.

Reply:
fixed